packages/dashboard/src/lib/repository.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  getBitbucketCloudBranchURL,
+  getBitbucketCloudCommitURL,
+  handleBitbucketCloudSshURL,
+} from '@sorry-cypress/dashboard/lib/bitbucketCloud';
+import {
+  getGithubBranchURL,
+  getGithubCommitURL,
+  handleGithubSshURL,
+} from '@sorry-cypress/dashboard/lib/github';
+
+export const getCommitURL = (repo: string, sha: string) =>
+  repo.includes('bitbucket.org')
```

Comment:
I was expecting this code not to be specific to bitbucket since it's in repository.ts file

Reply:
also, just a bit sad that it will just work with bitbucket cloud. pretty sure its the same logic needed for on prem version

Reply:
@FilipSwiatczak yeah but a cleaner way of doing it would be, each implementation should provide one canHandle method and we create an array with all supported implementation (in repository.ts) and we iterate through them til we find supported one.

@agoldis are you okay with those changes ?